apps/judge/src/app/(main)/[year]/submission/action.ts

```diff
@@ -2,6 +2,19 @@
 
 import { randomUUID } from "node:crypto";
 import { prisma } from "@repo/database";
+import { 
+	verifySubmissionUserTeam, 
+	verifySubmissionUserDraft,
+	verifySubmissionUserSubmission } from "./auth-actions";
+import {
+	createDraftSchema,
+	updateDraftSchema,
+	createSubmissionFromDraftSchema,
+	completeDraftSchema,
+	updateDataSchema,
+	sendDataSchema,
+} from "./mutator-schemas";
+import { z } from "zod";
 
 export async function createDraft(
```

Comment:
I mean you could probably extract the types from your zod schema and just make this json input, but I digress. nit not worth fixing